### ISSUE TITLE
Main markers not spawning

### ISSUE THREAD

**Issue by StefArma** [LordGolias/antistasi#131]
Version: develop https://github.com/LordGolias/antistasi/commit/8c0ea9bdacffc86ebd00146e22bf80d16e31a147
Mod: CBA, TFAR
Multiplayer dedicated server
Faction: both against NATO and AAF

Bases, rerouces, outposts, seaports etc. not spawning in any units., only town patrols , roadblocks and mission related are spawning

**Comment by LordGolias** [comment]
Thanks Stef. Was there anything relevant in the log file?

**Comment by StefArma** [comment]
5 min i'll check .rpt

**Comment by StefArma** [comment]
21:48:19 "[AS] 2: new spawn 'as_seaport_2'"
21:48:19 "[AS] Server: scheduler_fnc_execute(AS_spawn_fnc_execute):"
21:48:19 "Worker 12: 90.5654"
21:48:19 Server: Object 12:173 not found (message Type_452)
21:48:19 Server: Object 12:178 not found (message Type_260)
21:48:19 Server: Object 12:183 not found (message Type_460)
21:48:19 Server: Object 12:182 not found (message Type_460)
21:48:19 "Worker 2: 136.371"
21:48:19 "Worker 13: 48.0577"
21:48:19 Server: Object 12:177 not found (message Type_460)
21:48:19 Server: Object 12:181 not found (message Type_460)
21:48:19 Server: Object 12:180 not found (message Type_460)
21:48:19 Server: Object 12:179 not found (message Type_460)
21:48:19 Server: Object 12:176 not found (message Type_460)
21:48:19 Server: Object 12:175 not found (message Type_132)
21:48:19 Server: Object 12:174 not found (message Type_260)
21:48:19 Server: Object 12:185 not found (message Type_454)
21:48:19 Server: Object 12:186 not found (message Type_114)
21:48:19 Server: Object 12:184 not found (message Type_93)
21:48:19 "[AS] 2: new spawn 'as_resource_4'"
21:48:25 Performance warning: SimpleSerialization::Read 'resources' is using type of ,'TASK' which is not optimized by simple serialization, falling back to generic serialization, use generic type or ask for optimizations for these types
21:48:25 Setting invalid pitch 0.0000 for O Alpha 1-6:3 REMOTE
21:48:25 Performance warning: SimpleSerialization::Read 'resources' is using type of ,'TASK' which is not optimized by simple serialization, falling back to generic serialization, use generic type or ask for optimizations for these types
21:48:25 Performance warning: SimpleSerialization::Read 'resources' is using type of ,'TASK' which is not optimized by simple serialization, falling back to generic serialization, use generic type or ask for optimizations for these types
21:48:25 Setting invalid pitch 0.0000 for O Alpha 2-1:7 REMOTE
21:48:25 Performance warning: SimpleSerialization::Read 'resources' is using type of ,'TASK' which is not optimized by simple serialization, falling back to generic serialization, use generic type or ask for optimizations for these types
21:48:26 Performance warning: SimpleSerialization::Read 'resources' is using type of ,'TASK' which is not optimized by simple serialization, falling back to generic serialization, use generic type or ask for optimizations for these types
21:48:26 Setting invalid pitch 0.0000 for O Alpha 2-5:3 REMOTE
21:48:26 Performance warning: SimpleSerialization::Read 'resources' is using type of ,'TASK' which is not optimized by simple serialization, falling back to generic serialization, use generic type or ask for optimizations for these types
21:48:26 Server: Object 12:451 not found (message Type_109)
21:48:26 Performance warning: SimpleSerialization::Read 'resources' is using type of ,'TASK' which is not optimized by simple serialization, falling back to generic serialization, use generic type or ask for optimizations for these types
21:48:46 "[AS] Error: Client-only script AS_fnc_localCommunication called"
21:49:30 "[AS] 2: new spawn 'aaf_road_patrol_76'"
21:49:30 "[AS] Server: scheduler_fnc_execute(AS_spawn_fnc_execute):"

Nothing relevant to me

**Comment by StefArma** [comment]
Also, used a mortar nearby petros

i got this in .rpt
14:43:02 "[AS] Server: scheduler_fnc_execute(AS_mission_fnc_createDefendHQ):"

however no enemies spawned.

**Comment by StefArma** [comment]
Version: Develop: https://github.com/LordGolias/antistasi/commit/d8d75a59eb7130418757fa755a55716817866bec
Dedicated Server
Mods: None
Faction: against west

Still happening, factory, outpost etc. don't spawn regardless their function is assigned correctly to worker etc. 

"[AS] 2: new spawn 'as_outpost_13'"
11:44:56 "[AS] Server: scheduler_fnc_execute(AS_spawn_fnc_execute):"
11:44:56 "Worker 3: 91.8734"
11:44:56 "Worker 2: 136.562"
11:44:56 "Worker 4: 43.7743"
11:44:57 "[AS] 2: new spawn 'as_outpost_14'"
11:44:57 "[AS] Server: scheduler_fnc_execute(AS_spawn_fnc_execute):"
11:44:57 "Worker 3: 91.8734"
11:44:57 "Worker 2: 136.562"
11:44:57 "Worker 4: 43.7743"
11:44:57 "[AS] 2: new spawn 'as_outpost_15'"
11:44:57 "[AS] Server: scheduler_fnc_execute(AS_spawn_fnc_execute):"
11:44:57 "Worker 3: 91.8734"
11:44:57 "Worker 2: 136.562"
11:44:57 "Worker 4: 43.7743"

**Comment by LordGolias** [comment]
Is it possible to have the other worker's log? I suspect the worker is failing for some reason, and this is causing the spawn not to start. Having the worker's log will help understand this

**Comment by StefArma** [comment]
Actually i can't for headlessclint, it never display me log... anyway i can try disable headlessclient and see. Testing

**Comment by StefArma** [comment]
True, when Headlessclient is offline, enemies spawn, if i reconnect it they don't spawn anymore. Just outpost, factory etc.. because roadblocks and kavala spawns correctly

Nothing on headless .rpt

**Comment by StefArma** [comment]
Extra information:
- Headlessclient connected
- Port to Kavala
- Headlessclient disconnect

factory and outpost still don't spawn regardless they should be reassigned
11:43:48 Performance warning: SimpleSerialization::Read 'resources' is using type of ,'TASK' which is not optimized by simple serialization, falling back to generic serialization, use generic type or ask for optimizations for these types
11:44:31 Server: Object 6:141 not found (message Type_118)
11:44:31 Server: Object 6:140 not found (message Type_353)
11:44:31 No speaker given for 
11:44:35 Server: Object 6:149 not found (message Type_93)
11:44:35 Error: Object(6 : 149) not found
11:45:02 Client: Remote object 6:2 not found
11:45:02 "arma3/BIS_fnc_log: HandleDisconnect : O Alpha 1-6:1"
11:45:02 A null object passed as a target to RemoteExec(Call) 'bis_fnc_objectvar'
11:45:03 Warning: Cleanup player - person 6:104 not found
11:45:05 Warning: Cleanup player - person 6:104 not found
11:45:13 No speaker given for 
11:45:13 No speaker given for 
11:46:40 Client: Remote object 5:2 not found
11:46:40 "arma3/BIS_fnc_log: HandleDisconnect : HC1"

I try the test again paying attention on .rpt who are places assigned to

**Comment by LordGolias** [comment]
Ok, can you describe what exact steps you take in order to reproduce this? I am not being able to reproduce it, so it is hard for me to debug :(

**Comment by MSchietzsch** [comment]
Got pretty much the same problem. But it seems civ are spawning... 
Arma Version: 1.82.144710
Server is a dedicated (running on windows)
3x Headles, also on win
Antistasi Version: v 1.1.3

Mods:
CBA
RHSAFRF
RHSGREF
RHSSAF
RHSUSAF
TFAR
WotP_ACE3

[HC3_arma3server_x64_2018-07-06_19-13-43.rpt.log](https://github.com/LordGolias/antistasi/files/2171487/HC3_arma3server_x64_2018-07-06_19-13-43.rpt.log)
[SERVER_arma3server_x64_2018-07-06_19-29-47.rpt.log](https://github.com/LordGolias/antistasi/files/2171488/SERVER_arma3server_x64_2018-07-06_19-29-47.rpt.log)
[HC1_arma3server_x64_2018-07-06_19-13-33.rpt.log](https://github.com/LordGolias/antistasi/files/2171489/HC1_arma3server_x64_2018-07-06_19-13-33.rpt.log)
[HC2_arma3server_x64_2018-07-06_19-13-38.rpt.log](https://github.com/LordGolias/antistasi/files/2171490/HC2_arma3server_x64_2018-07-06_19-13-38.rpt.log)

**Comment by MSchietzsch** [comment]
oh wait this repo is dead, isn't it?

**Comment by LordGolias** [comment]
I believe that this was fixed by the changes of 2799138
